secure/storage/vault/src/lib.rs

```diff
@@ -28,8 +28,8 @@ pub mod fuzzing;
 const MAX_NUM_KEY_VERSIONS: u32 = 4;
 
 /// Default request timeouts for vault operations.
-const DEFAULT_CONNECTION_TIMEOUT_MS: u64 = 10_000;
-const DEFAULT_RESPONSE_TIMEOUT_MS: u64 = 10_000;
+const DEFAULT_CONNECTION_TIMEOUT_MS: u64 = 100;
```

Comment:
this seems like some crazy low defaults, why aren't we comfortable waiting for 1 second? spin harder? fail faster?

Reply:
Why are these crazy low? It takes <10ms to create a TCP connection to the Vault LB, so 100ms connect timeout is >10x the expected time. And the p99 for Vault requests is usually <100ms, so 1s request timeout is also >10x the expected time. For comparison the recommended timeout for SR in FB is p99 * 1.5. The point is to fail fast.

Reply:
I was pretty sure I saw RTT latency of around 100 ms for safety rules. Assuming it is 10ms, I agree, assuming that the 100ms is accurate I am concerned that this is too aggressive.

Reply:
100ms for the TCP connect or for the whole Vault request?